tests/environment-edit.bats

```diff
@@ -0,0 +1,152 @@
+#! /usr/bin/env bats
+# -*- mode: bats; -*-
+# ============================================================================ #
+#
+# Test flox edit
+#
+# ---------------------------------------------------------------------------- #
+
+load test_support.bash
+
+# ---------------------------------------------------------------------------- #
+
+# Helpers for project based tests.
+
+project_setup() {
+  # Test manifests
+  export ORIGINAL_MANIFEST_CONTENTS=$(cat << EOF
+{
+  context,
+  system,
+  ...
+}:
```

Comment:
Not important but don't think we need these? 
```suggestion
{}:
```

Reply:
I just copied and pasted the `flox.nix` that was generated by `init`.